js/index.js

```diff
@@ -13,4 +13,32 @@ skills.forEach((skill) => {
     const skillItem = document.createElement("li")
     skillItem.innerText = skill
     skillsList.appendChild(skillItem)
-}) 
\ No newline at end of file
+}) 
+
+/* message form submit */
+const messageForm = document.getElementsByName('leave_message')[0];
+messageForm.addEventListener('submit', (e) =>{
+    e.preventDefault();
+    const name = e.target.name.value;
+    const email = e.target.email.value;
+    const message = e.target.message.value;
+    console.log(name, email, message);
+
+    const messageSection = document.getElementById('messages');
+    const messageList = messageSection.querySelector('ul');
+    const newMessage = document.createElement('li');
+    newMessage.innerHTML = `<a href="mailto:${email}">${name} </a><span>wrote: ${message}</span>`;
```

Comment:
```suggestion
    newMessage.innerHTML = `<a href="mailto:${email}">${name}</a> <span>wrote: ${message}</span>`;
```

With a trailing space inside the `<a>`, the underline extends too far.